src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -551,7 +599,24 @@ private IVariableInformation[] ExpandVisualized(IVariableInformation variable)
                             arrayExpr.EnsureChildren();
                             if (arrayExpr.CountChildren != 0)
                             {
-                                children.AddRange(arrayExpr.Children);
+                                uint currentIndex = 0;
+                                var pvwVariable = variable as PaginatedVisualizerWrapper;
```

Comment:
You can use the `is` syntax here

```suggestion
                                if (variable is PaginatedVisualizerWrapper pvwVariable )
                                {
                                   currentIndex =pvwVariable.StartIndex;
                                }
```